src/AddTodoForm.jsx

```diff
@@ -0,0 +1,16 @@
+function AddTodoForm () {
+    return (
+        <>
```

Comment:
react fragment unnecessary if you are returning JSX with one top level tag

Reply:
Okay I'll remove it. 

function AddTodoForm () {
    return (
        <form>
            <label>Title</label>
            <input type="text" id="todoTitle" htmlFor="todoTitle"></input>
            <button type="submit">Add</button>
        </form>
    )
}